src/Websites/Websites/help/New-AzWebAppManagedCertificate.md

```diff
@@ -0,0 +1,163 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Websites.dll-Help.xml
+Module Name: Az.Websites
+online version:https://docs.microsoft.com/en-us/powershell/module/az.websites/new-azwebappmanagedcertificate
+schema: 2.0.0
+---
+
+# New-AzWebAppManagedCertificate
+
+## SYNOPSIS
+Creates an App service managed certificate for an Azure Web App. 
+
+## SYNTAX
+
+```
+New-AzWebAppManagedCertificate [-ResourceGroupName] <String> [-WebAppName] <String> [[-Slot] <String>]
+ [-HostName] <String> [-AddCertBinding] [[-SslState] <SslState>] [-DefaultProfile <IAzureContextContainer>]
```

Comment:
Change this to AddBinding

Reply:
did it as per Design review. will change it as you suggested.